src/ripple/rpc/impl/RPCHelpers.h

```diff
@@ -114,10 +114,34 @@ parseRippleLibSeed(Json::Value const& params);
 std::pair<PublicKey, SecretKey>
 keypairForSignature(Json::Value const& params, Json::Value& error);
 
+/**
+ * API version numbers used in API version 1
+ */
 extern beast::SemanticVersion const firstVersion;
 extern beast::SemanticVersion const goodVersion;
 extern beast::SemanticVersion const lastVersion;
 
+/**
+ * API version numbers used in later API versions
+ *
+ * Requests with a version number in the range
+ * [APIVersionRangeLow, APIVersionRangeHigh] are supported.
+ *
+ * Network Requests without explicit version numbers use
+ * APIVersionIfUnspecified.
+ * Command line Requests use APIVersionCommandLine.
+ */
+
+constexpr unsigned APIInvalidVersion = 0;
+constexpr unsigned APIFirstVersion = 1;
```

Comment:
We only use this here to set `APIVersionIfUnspecified`. I'd remove it, and just set `APIVersionIfUnspecified` to be equal to `APIVersionSupportedRangeLow`.

Reply:
-- I will remove "APIFirstVersion". As you said, it is only for setting (actually documenting) APIVersionIfUnspecified. 
-- APIVersionIfUnspecified should always be 1. Since all the RPC requests with a version >= 2 must explicitly specify the version in the requests. If agreed, I will add a comment in code to explain.

Reply:
I think that's fine. Thanks!

Reply:
Removed `APIFirstVersion`.
`APIVersionIfUnspecified` is set to 1.